tests/gocase/unit/search/search_test.go

```diff
@@ -88,12 +104,30 @@ func TestSearch(t *testing.T) {
 			fields := res.Val().([]interface{})[2].([]interface{})
 			if fields[0] == "a" {
 				require.Equal(t, "x,z", fields[1])
-				require.Equal(t, "b", fields[2])
-				require.Equal(t, "22", fields[3])
+				if fields[2] == "c" {
+					require.Equal(t, "12.000000, 13.000000, 14.000000", fields[3])
+					require.Equal(t, "b", fields[4])
+					require.Equal(t, "22", fields[5])
+				} else if fields[2] == "b" {
+					require.Equal(t, "22", fields[3])
+					require.Equal(t, "c", fields[4])
+					require.Equal(t, "12.000000, 13.000000, 14.000000", fields[5])
+				} else {
+					require.Fail(t, "not started with c or b")
+				}
```

Comment:
maybe we can read them to a map and then do some check so that we don't need to care the order.

Reply:
good idea! I've pushed new code change